package/boot/uboot-envtools/files/qualcommax_ipq807x

```diff
@@ -12,7 +12,8 @@ dynalink,dl-wrx36|\
 netgear,rax120v2|\
 netgear,wax218|\
 netgear,wax620|\
-netgear,wax630)
+netgear,wax630|\
+yuncore,ax880)
 	idx="$(find_mtd_index 0:appsblenv)"
 	[ -n "$idx" ] && \
 		ubootenv_add_uci_config "/dev/mtd$idx" "0x0" "0x40000" "0x20000" "2"
```

Comment:
Are you sure it is correct? The size of this partition is only 64k.
https://github.com/openwrt/openwrt/blob/6e5edec159e2dc468607a5b9179f722857a38421/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-ax880.dts#L196-L197

Reply:
Yeah, its clearly not correct as this one is for 2 blocks on NAND

Reply:
Sorry. Wrong block. 
Correct one is:
/dev/mtd${idx} 0x10000 0x10000

Will fix.

Reply:
fixed

Reply:
As an alternative, please try adding `compatible = "u-boot,env";` property to the device tree u-boot-env partition.

Reply:
ok. dt way is better i think.

Reply:
That is for the in-kernel parser, it doesn't provide configuration for fw_print/setenv

Reply:
u-boot envtools support this feature since v2024.01. I've tested it on ath79 target. https://github.com/openwrt/openwrt/commit/d7d94a8d91bc1dc9dcedbe004021509871acac24

Reply:
Oh, I did not know that, latest I saw was that it can finally autodetect NAND blocks.
But if it can find the compatible, then we can get rid of most of the entries.

Reply:
```sh
root@ru-ax880:/etc# ls /etc/fw_env.config
ls: /etc/fw_env.config: No such file or directory
root@ru-ax880:/etc# fw_printenv stdin
stdin=serial@78B3000
root@ru-ax880:/etc#
```

it's working. @DragonBluep thanks.